tests/contrib/starlette/test_starlette.py

```diff
@@ -3,35 +3,53 @@
 
 import httpx
 import pytest
+import sqlalchemy
+
 import ddtrace
+from ddtrace import Pin
 from ddtrace import config
-from ddtrace.contrib.starlette import patch, unpatch
+from ddtrace.contrib.starlette import patch as starlettePatch, unpatch as starletteUnpatch
+from ddtrace.contrib.sqlalchemy import patch as sqlPatch, unpatch as sqlUnpatch
 from ddtrace.propagation import http as http_propagation
+
+
 from starlette.testclient import TestClient
-from tests import override_http_config
+from tests import override_http_config, snapshot
 from tests.tracer.test_tracer import get_dummy_tracer
+
 from app import get_app
 
 
 @pytest.fixture
-def tracer():
+def engine():
+    DATABASE_URL = "sqlite:///test.db"
+    sqlPatch()
+    engine = sqlalchemy.create_engine(DATABASE_URL)
```

Comment:
Does this have to be done on each test invocation? It seems like we can just do this once at the top of the file:

```python
engine = sqlalchemy.create_engine("sqlite:///test.db")
```